modules/andes-core/broker/src/main/java/org/wso2/andes/store/cassandra/CQLBasedMessageStoreImpl.java

```diff
@@ -853,4 +854,43 @@ private ResultSet execute(Statement statement, String task) throws AndesExceptio
             throw new AndesException("Error occurred while " + task, e);
         }
     }
+
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public void storeRetainedMessages(List<AndesMessage> retainList) throws AndesException {
+        // TODO: implement this method
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public List<String> getAllRetainedTopics() throws AndesException {
+        // TODO: implement this method
+        return  null;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public AndesMessageMetadata getRetainedMetaData(String destination) throws AndesException {
+        // TODO: implement this method
+        return  null;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public Map<Integer, AndesMessagePart> getRetainedContentParts(long messageID) throws AndesException {
+        // TODO: implement this method
+        return  null;
+    }
+
```

Comment:
Explain why these methods are not implemented. If they suppose to return null, remove todo.